dts/bindings/i2s/st,stm32-i2s.yaml

```diff
@@ -25,3 +25,10 @@ properties:
 
   pinctrl-names:
     required: true
+
+  mck-enabled:
+    type: boolean
+    description: |
+      Master Clock Output function.
+      The designated phandle with the supported alternate function pin must
+      be listed in the pinctrl-0 array when enabling this property.
```

Comment:
Would you mind re-phrasing this in a more simple way: "a mck pin must be listed..." ?